### ISSUE TITLE
HUD not working

### ISSUE THREAD

**Issue by cookieasd** [Mytherin/Tibialyzer#272]
so since tibia 11 came out the HUD stopped working for me before everything was working fine and i kind of stopped playing tibia for awhile thats why its kind of a late post i tried running them both as admin and i have the latest update for both of them

**Comment by fclaussen** [comment]
Which hud is not working? They work fine to me

**Comment by cookieasd** [comment]
all of them thats the wierd thing it started only with the mana and health and now just everything

**Comment by Mytherin** [comment]
Are you using the latest version of Tibialyzer?

**Comment by cookieasd** [comment]
yes i did the update data base and i downloaded the new address the version i have is 1.4.2

**Comment by cookieasd** [comment]
the mana and health are working right now but the friend list one still not working is it something i am doing wrong with it?
